applications/launchpad_v2/src/containers/MiningContainer/index.tsx

```diff
@@ -1,20 +1,37 @@
-import { useDispatch } from 'react-redux'
+import { useDispatch, useSelector } from 'react-redux'
+import Switch from '../../components/Switch'
+
+import Text from '../../components/Text'
+import SvgSun from '../../styles/Icons/Sun'
+import SvgMoon from '../../styles/Icons/Moon'
 
 import { setTheme } from '../../store/app'
+import { selectTheme } from '../../store/app/selectors'
 
 /**
  * @TODO move user-facing text to i18n file when implementing
  */
 
 const MiningContainer = () => {
   const dispatch = useDispatch()
+  const currentTheme = useSelector(selectTheme)
```

Comment:
what's the difference here between this and `useTheme` from `styled-components`?